frontend/src/main/components/Users/UsersTable.js

```diff
@@ -29,11 +31,75 @@ const columns = [
         id: 'admin',
         accessor: (row, _rowIndex) => String(row.admin) // hack needed for boolean values to show up
     },
+    {
+        Header: 'Instructor',
+        id: 'instructor',
+        accessor: (row, _rowIndex) => String(row.instructor) // hack needed for boolean values to show up
+    }
 ];
 
-export default function UsersTable({ users }) {
+// function cellToAxiosParamsToggleAdmin(cell){
```

Comment:
What is the purpose of this commented out code?

Please see: 

https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

Reply:
I first tried to define `cellToAxiosParamsToggleAdmin` and `cellToAxiosParamsToggleInstructor` inside UsersTable.js, but later moved them into UsersTableUtils. Thanks for the doc! I will remove them now